### PR TITLE
Uploading OTDB_Protocol

### PR DESCRIPTION
**Pull request by Aroke-Anna8**
Hi Dr. Martin,
Hoping I did this right. Please let me know if there are any changes to be done.
Thank you.
Anna.

### CODE REVIEW

File: OTDB_Protocol.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+**Beneficence**
+
+The intention of this data bank is to provide data for future research and quality improvement in the field of trauma and trauma care. This research will in turn hopefully improve trauma outcomes by improving trauma patient care and trauma patient management.
+
+**Justice**
+
+All patients included in this study were treated regardless of age, sex, or prior medical history. Patients received the same level of care whether they choose to participate in the data bank or not.
+
+**Data management**
```
Comment (martingerdin) [inline OTDB_Protocol.md]: This should not be a subheading under ethical considerations but separate.

File: OTDB_Protocol.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+One of the most important components of trauma systems is a trauma registry, or a trauma data bank (9). The data bank is used to track trauma patients and their outcomes to facilitate trauma surveillance and research. Towards Improved Trauma Care Outcomes (TITCO), Swedish Trauma Registry (SweTrau) and National Trauma Data Bank (NTDB) are examples of registries currently in use. All of these have been used in trauma research, contributing to clinical guidelines and improved patient care – and ultimately to reduced trauma mortality and morbidity (10-12).
+
+This project proposes to implement a trauma data bank at [REPLACE WITH CENTRE NAME], with the aim to collect the absolute latest available trauma data for use in research or institutional quality improvement programs.
+
+## Methods
+
+#### Design
```
Comment (martingerdin) [inline OTDB_Protocol.md]: Use ### instead

File: OTDB_Protocol.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+Data bank implementation.
+
+#### Setting
+
+Hospitals that receive trauma patients.
+
+#### Participants
+**Eligibility criteria**
```
Comment (martingerdin) [inline OTDB_Protocol.md:37]: Use #### instead

**Review by martingerdin — CHANGES_REQUESTED** [review]
Thanks for this Anna! Please revise the heading levels according to the comments.

File: OTDB_Protocol.md (hunk 1 above)
Comment (Aroke-Anna8) [reply OTDB_Protocol.md]: Noted and made the necessary changes

File: OTDB_Protocol.md (hunk 3 above)
Comment (Aroke-Anna8) [reply OTDB_Protocol.md:37]: Completed.

**Pull request merged**